### PR TITLE
Tech Component

### PR DESCRIPTION
**Pull request by alejobh**
## Summary
The purpose of this branch is to add the Tech Component.

## Screenshots

image

## Trello Card
https://trello.com/c/WNg7hyka

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -10,5 +10,15 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>
+  <div class="text-container">
```
Comment (henryzarza) [inline maquetado/index.html]: This class name it's not very descriptive, you change maybe for `techs-container` or other class name.

File: maquetado/index.html [hunk 2]
```diff
@@ -10,5 +10,15 @@
...
+      <div class="tech-info text-center">
+        <h3 class="title-medium m-top-1 m-bottom-1">ReactJS</h3>
+        <span class="title-medium-bold m-bottom-2">Release: 2013</span>
+        <span class="title-medium">React is a declarative, component-based Javascript library for building user interfaces.</span>
+      </div>
+    </div>
+  </div>
 </body>
-</html>
+</html>
```
Comment (henryzarza) [inline maquetado/index.html]: Add EOF (end of file).

File: maquetado/scss/margins.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  margin-bottom: 20px;
+}
+
+.m-top-1 {
+  margin-top: 10px;
+}
+
+.m-top-2 {
+  margin-top: 20px;
+}
```
Comment (henryzarza) [inline maquetado/scss/margins.scss]: Add EOF

File: maquetado/scss/styles.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
+@import 'titles';
+@import 'variables';
+@import 'tech';
+
+body {
+	font-family: $font-family;
+}
```
Comment (henryzarza) [inline maquetado/scss/styles.scss]: Add EOF

File: maquetado/scss/styles.scss [hunk 5]
```diff
@@ -0,0 +1,7 @@
+@import 'titles';
+@import 'variables';
+@import 'tech';
+
+body {
+	font-family: $font-family;
```
Comment (henryzarza) [inline maquetado/scss/styles.scss]: Remember the size the tab is 2

File: maquetado/scss/tech.scss [hunk 6]
```diff
@@ -0,0 +1,58 @@
+@import 'variables';
+@import 'margins';
+
+$tech-height: 390px;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: You set all this variables in the file of `variables`

File: maquetado/scss/tech.scss [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+$tech-width: 270px;
+$border-radius: 5px;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Remember order by alphabetically

File: maquetado/scss/tech.scss [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+  background-color: $green;
+  border-radius: $border-radius;
+  display: flex;
+  flex-direction: column;
+  height: $tech-height;
+  position: relative;
+  width: $tech-width;
+  
+  &::before {
+    border: 1px solid #BED23C;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: This color must be in the file of `colors` and here you should import it

File: maquetado/scss/tech.scss [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+    margin: 15px 15px;
+    position: absolute;
+    top: $position;
+    width: $tech-width;
+  }
+}
+
+.tech-image{
+  width: $tech-image-size;
+  height: $tech-image-size;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Is not a good practice assign a height static to the image, you remember that the size is smaller and order the properties by alphabetically

File: maquetado/scss/tech.scss [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+.tech-info {
+  padding: 0 50px;
+}
+
+.text-center {
+  display: block;
+  text-align: center;
+}
+
+.title-medium ,
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Delete the space between the text and the `,`

**Review by henryzarza — CHANGES_REQUESTED** [review]

File: maquetado/scss/styles.scss (hunk 5 above)
Comment (henryzarza) [inline maquetado/scss/styles.scss]: Remember the identation (2)

**Review by henryzarza — DISMISSED** [review]

File: maquetado/scss/styles.scss (hunk 5 above)
Comment (braiandickson) [reply maquetado/scss/styles.scss]: @alejobh Use only 2 space for indentation.

File: maquetado/scss/tech.scss [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+    height: $tech-height;
+    left: $tech-back-position;
+    margin: 15px 15px;
+    position: absolute;
+    top: $tech-back-position;
+    width: $tech-width;
+  }
+}
+
+.tech-image{
```
Comment (braiandickson) [inline maquetado/scss/tech.scss]: Add a space before the {

**Review by braiandickson — CHANGES_REQUESTED** [review]

**Review by braiandickson — DISMISSED** [review]

**Review by braiandickson — APPROVED** [review]

**Pull request merged**